### PR TITLE
Create accessing-git-lfs.md

### PR DESCRIPTION
**Pull request by icarusdust**
How to access Git LFS

### CODE REVIEW

File: content/knowledge-base/accessing-git-lfs.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+description: How to use git lfs with codemagic
```
Comment (helinanever) [inline content/knowledge-base/accessing-git-lfs.md:2]:
```suggestion
description: How to use Git LFS with Codemagic
```

File: content/knowledge-base/accessing-git-lfs.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+title: Accessing Git LFS
+weight: 9
+aliases:
+  - '../custom-scripts/accessing-git-lfs'
```
Comment (helinanever) [inline content/knowledge-base/accessing-git-lfs.md:6]: no aliases are needed for this page

File: content/knowledge-base/accessing-git-lfs.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+---
+
```
Comment (helinanever) [inline content/knowledge-base/accessing-git-lfs.md:8]: unnecessary empty row

File: content/knowledge-base/accessing-git-lfs.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+Git Large File Storage (LFS) replaces large files such as audio samples, videos, datasets, and graphics with text pointers inside Git, while storing the file contents on a remote server like GitHub.com or GitHub Enterprise. More info about Git LFS can be found [here]("https://git-lfs.github.com/").
+
+In order to access Git LFS (an open-source Git extension for versioning large files) during a build, add `git lfs install --skip-smudge` in a post-clone script.
```
Comment (helinanever) [inline content/knowledge-base/accessing-git-lfs.md:12]: maybe a couple of words where the script needs to be added in flutter workflow editor vs yaml?

**Review by helinanever — CHANGES_REQUESTED** [review]
I've left some comments

**Comment by icarusdust** [comment]
Changes made!

**Review by nevercode-kevin — APPROVED** [review]
LGTM

**Pull request merged**